ds3-sdk/src/main/java/com/spectralogic/ds3client/commands/GetBucketRequest.java

```diff
@@ -52,7 +53,7 @@ public GetBucketRequest withNextMarker(final String nextMarker) {
      */
     public GetBucketRequest withPrefix(final String prefix) {
         this.prefix = prefix;
-        this.getQueryParams().put("prefix", prefix);
+        this.getQueryParams().put("prefix", UrlEscapers.urlFragmentEscaper().escape(prefix));
```

Comment:
I don't see a unit test for this line.  We probably need one to make sure the escaping behavior that we assume needs to be here, is actually needed.